tests/cli/test_cli_options.py

```diff
@@ -544,11 +555,11 @@ def main(args):
     '''
     Check if the binary was build as Release-type
     
-    TODO: find a check for Windows,
-          for now rely on the ../../build/Release/ location
+    TODO: fix this for multi-configuration generators
```

Comment:
Note: Checking the OS type here is wrong. This check obviously exists because VS is a multi-configuration generator, which means the build type cannot be determined from the cache. However, VS is not the only multi-configuration generator (XCode is another), nor is it the only generator available on Windows.

There *is* a way to check if a build is using a multi-configuration generator, but I forget the variable offhand, so have left it alone for now, other than tweaking the comment. If there is a strong preference to fix it now, however, I can go dig up the right way to do this...

Reply:
I've addressed this now, partly due to needing to handle it better in the `CMakeLists.txt` when the test and target are set up, in order to not do so for single-configuration generators to avoid CI failures.